docs/.vuepress/config.js

```diff
@@ -45,8 +45,10 @@ module.exports = {
           title: 'Resources',
           children: [
             {
+              // TODO(creachadair): Figure out how to make this per-branch.
+              // See: https://github.com/tendermint/tendermint/issues/7908
               title: 'RPC',
-              path: 'https://docs.tendermint.com/master/rpc/',
+              path: 'https://docs.tendermint.com/v0.35/rpc/',
```

Comment:
There isn't a variable name you can use to see the current version?

Reply:
There may be—but I don't know how to interpolate that into these static configs. There may wind up being a simple solution, but so far it looks like we might need to add a plugin. I figured meanwhile we can keep the docs working.